functests/2_performance_update/updating_profile.go

```diff
@@ -269,6 +270,10 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 			kblcfg, err := nodes.ExecCommandOnNode(chkKubeletConfig, newCnfNode)
 			Expect(err).ToNot(HaveOccurred(), fmt.Sprintf("failed to execute %s", chkKubeletConfig))
 			Expect(kblcfg).NotTo(ContainSubstring("reservedSystemCPUs"))
+
+			irqbal, err := nodes.ExecCommandOnNode(chkIrqbalance, newCnfNode)
+			Expect(err).ToNot(HaveOccurred(), fmt.Sprintf("failed to execute %s", chkIrqbalance))
+			Expect(irqbal).NotTo(ContainSubstring("ffffffff,fffffffe"))
```

Comment:
"ffffffff,fffffffe" - is only for scenario with  profile parameter  **reserved: "0"**
So, this test may not work correctly on cluster with different "reserved" value. 

How about adding calculation of this value based on profile setting?

Reply:
Agree it is good to have it this way.

but before that, it needs to be investigated why we see the following in ci/prow/e2e-gcp which probably means that on reverting the label the file is being removed in which case we would need to check if the file exists.

```
  out=cat: /etc/sysconfig/irqbalance: No such file or directory
```

Reply:
Missed to add _/rootfs_ prefix for the no such file error. Fixed it.